pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -456,6 +470,18 @@ private static IEnumerable<Form> OpenForms
             return null;
         }
 
+        public static Form FindOpenForm(Type formType) 
+        {
+            foreach (var form in OpenForms)
+            {
+                if (((formType.IsInstanceOfType(form) || formType.DeclaringType != null && formType.DeclaringType.IsInstanceOfType(form))) && form.Created)

```

Comment:
Why do we need the "formType.DeclaringType" here?
Which form class are we looking for?

Reply:
Because tests often use a nested class when calling PauseForScreenShot like here:
`PauseForScreenShot<ViewEditor.ChooseColumnsView>("Edit Report form", 4);`
The nested class is usually one of the tabs in the parent form, so when we look for an open window we need the declaring class which is an actual form containing this tab.